test/lib/bamboo/adapters/smtp_adapter_test.exs

```diff
@@ -675,6 +675,61 @@ defmodule Bamboo.SMTPAdapterTest do
     assert_configuration(bamboo_config, gen_smtp_config)
   end
 
+  test "email looks fine when they have non-ASCII characters in subject, from and to" do
+    bamboo_email =
+      new_email(
+        from: {"Awesome Person 😎", "awesome@person.co"},
```

Comment:
I would suggest using a `.local` extension, we never know 😅

Reply:
Yeah, good point. 😅